app/javascript/Pages/Login.jsx

```diff
@@ -16,6 +17,9 @@ class Login extends Component {
     const handleSubmit = (e) => {
       e.preventDefault();
 
+      let username = document.getElementById("username");
```

Comment:
**issue (blocking)**: in react you should not get data from the DOM like this. Either add an event handler to onChange in each input or use a two-way binding system like https://reactgo.com/two-way-data-binding-react/#two-way-data-binding-in-react-hooks